Runtime/src/main/java/com/rakuten/tech/mobile/perf/runtime/internal/Util.java

```diff
@@ -1,18 +1,28 @@
 package com.rakuten.tech.mobile.perf.runtime.internal;
 
 import android.content.Context;
+import android.content.pm.PackageInfo;
 import android.content.pm.PackageManager;
+import android.content.pm.Signature;
 import android.os.Bundle;
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
+import android.support.annotation.VisibleForTesting;
+import java.io.ByteArrayInputStream;
+import java.security.cert.CertificateFactory;
+import java.security.cert.X509Certificate;
+import javax.security.auth.x500.X500Principal;
 
 /**
  * Shared functions for Relay SDKs
  */
-class Util {
+@VisibleForTesting(otherwise = VisibleForTesting.PACKAGE_PRIVATE)
+public class Util {
```

Comment:
please also use [`@RestrictTo(LIBRARY)`](https://developer.android.com/reference/android/support/annotation/RestrictTo.html)

- [ ] add annotation

Reply:
Your were right, VisibleForTesting is any for validation. RestrictTo is necessary. Added.